docs/topics/proc_showing-debug-logging-on-openshift.adoc

```diff
@@ -0,0 +1,116 @@
+
+[#showing-debug-logging-on-openshift]
+= Showing Debug Logging on OpenShift
+
+Start your application and interact with it to see the debugging statements in OpenShift.
+
+.Prerequisites
+
+* A Maven-based application
+ifndef::wf-swarm[with debug logging enabled]
+* The `oc` CLI client installed and authenticated
+
+
+.Procedure
+
+ifndef::wf-swarm[]
+. Deploy your application to OpenShift:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:deploy -Popenshift
+----
+endif::[]
+
+ifdef::wf-swarm[]
+. Set the `swarm.logging` property to `DEBUG` when deploying your application to OpenShift:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:deploy -Popenshift -Dswarm.logging=DEBUG
```

Comment:
Did you verify? I don't this will do what you think it will do.

Reply:
I'll check; why do you think it won't work, though? Won't the `swarm.logging` property be used in the deployment?

Reply:
I'm 99.9% sure the system property won't propagate from your local machine to the pod inside the OpenShift cluster.

Reply:
I see, I'll use to the YAML configuration then. Thanks.